Update README to mention confidence intervals.md (#32)
open-spaced-repetition/fsrs-benchmark: README.md

Total number of reviews for evaluation: 697,851,710

> The values in the tables below are weighted means and their respective 99% confidence intervals.

Reviewer: ```suggestion
The following tables represent the weighted means and the 99% confidence intervals.
```

Author: Typo: respresent
